Added instructions to quickly get eye tracking dataset NVGaze working

ArmandB opened the pull request:
I don't think the issue tab is enabled on RITnet, so I wasn't able to create an issue, but I tried following the ILLIXR contributing instructions as much as possible.

abehroozi committed:
added instructions for how to quickly get eye tracking dataset NVGaze working

mhuzai commented on README.md:
````diff
@@ -15,6 +15,14 @@ RITnet is the winnning model of the OpenEDS Semantic Segmentation Challenge. If
 }
 ```
 
+Dataset:
````
Remove the heading and move the paragraph right after `Instructions:`.

mhuzai commented on README.md:
````diff
@@ -15,6 +15,14 @@ RITnet is the winnning model of the OpenEDS Semantic Segmentation Challenge. If
+sudo apt-get install ffmpeg
+ffmpeg -i 000001.jpg -vf scale=640:400 000001.png
+```
+
````
Here, say where to put the images (Semantic_Segmentation_Dataset/etc/etc).

mhuzai reviewed (commented):
Left a few minor comments.

ArmandB committed:
Update README.md

Removed dataset heading and moved it after "Instructions." Clarified where to put the images.

mhuzai reviewed (approved):
Thank you!
